### PR TITLE
 Zou22c - 10.1101/2022.11.17.516898 #721

### PR DESCRIPTION
**Pull request by zachandfox**
Zou22c - 10.1101/2022.11.17.516898 #721

### CODE REVIEW

**Comment by zachandfox** [comment]
CP infection dates and infection variant were not given, I chose '2021-01-01' and 'Unknown variant'

File: tables/subject_vaccines/zou22c-vacc.csv [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿ref_name,subject_name,vaccination_date_cmp,vaccination_date,vaccine_name,dosage,location,section
+Zou22c,Bivalent_CP,~,2021-01-01,BNT162b2,1,USA,NULL
```
Comment (KaimingTao) [inline tables/subject_vaccines/zou22c-vacc.csv:2]: The `vaccine_name` of subject `Bivalent_CP` is not right, please use `BNT162b2_BA.4/5_bivalent`

Comment (KaimingTao) [reply tables/subject_vaccines/zou22c-vacc.csv:2]: @zachandfox please use `inf` to indicate infection with vaccination. Please use `CP` for infection-only samples.

**Comment by KaimingTao** [comment]
> CP infection dates and infection variant were not given, I chose '2021-01-01' and 'Unknown variant'

No problem.

**Pull request merged**